example/test_problem/Hydro/CCSN/python_script/convert_progenitor_form.py

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env python
+#
+# -*- coding: utf-8 -*-
+#
+#  Purpose:
+#    Converting the progenitor file format (WH07, Heager2005, Sukhbold2015, and MESA)
+#    to the format that GAMER can use for CCSN test problems
+#
+
+import argparse
+import sys
+import numpy as np
+
+
+def convert_WHS(file):
+
```

Comment:
Remove this empty line. Same for `convert_MESA()` and `__name__ == "__main__"`

Reply:
I removed these empty lines.